test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/ValidationSummaryTagHelperTest.cs

```diff
@@ -312,14 +312,13 @@ public class ValidationSummaryTagHelperTest
             var validationSummaryTagHelper = new ValidationSummaryTagHelper(generator);
             var validationTypeName = typeof(ValidationSummary).FullName;
             var expectedMessage =
-                $@"The value of argument 'value' ({validationSummary}) is invalid for Enum type '{validationTypeName}'.
-Parameter name: value";
+                $@"The value of argument 'value' ({validationSummary}) is invalid for Enum type '{validationTypeName}'.";
```

Comment:
No need to change anything here. `ExceptionAssert` adds nothing if the test already works reliably. `[ReplaceCulture]` does the Right Thing<sup>:tm:</sup>.

Are you switching to `ExceptionAssert` beyond tests that fail in other cultures? If yes, either revert those changes or also get rid of `[ReplaceCulture]`. Don't double-fix locale-related issues.